src/channel.cr

```diff
@@ -1,5 +1,21 @@
 require "fiber"
 
+# A `Channel` enables concurrent communication between fibers.
+#
+#
```

Comment:
These suggestions are really tiresome, desired outcome should be:

```cr
# A `Channel` ...
#
# ```
# ...
# ```
```

i.e. with 1 empty line between the doc comment and example below.